rules/prefer-modern-dom-apis.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+const getDocsUrl = require('./utils/get-docs-url');
+
+const checkForReplaceChildOrInsertBefore = (context, node) => {
+	const getArgumentName = args => {
+		if (args.type === 'Identifier') {
+			return args.name;
+		}
+
+		return null;
+	};
```

Comment:
Move this helper function out of `checkForReplaceChildOrInsertBefore`. There may be a suitable util here: https://github.com/sindresorhus/eslint-plugin-unicorn/tree/master/rules/utils. Perhaps `is-method-named`?

Also, `args` is not a very good argument name. Perhaps `nodeArgument`?

Reply:
`is-method-named` would not help me, since i cannot say what the argument name will be.
There are multiple other rules with `const args = node.arguments;` and their own scoped helper functions, e.g: `prefer-node-remove` in the code already, but I am also fine with renaming it ;)